core/src/main/java/com/oath/oak/Chunk.java

```diff
@@ -17,27 +18,45 @@
 import java.util.concurrent.atomic.AtomicReference;
 import java.util.function.Consumer;
 
+import static com.oath.oak.NativeAllocator.OakNativeMemoryAllocator.INVALID_BLOCK_ID;
+import static com.oath.oak.UnsafeUtils.intsToLong;
+
 public class Chunk<K, V> {
 
     /*-------------- Constants --------------*/
 
-    static final int NONE = 0;    // constant for "no index", etc. MUST BE 0!
+    enum OFFSET {
+        NEXT(0), KEY_POSITION(3), VALUE_STATS(1), VALUE_POSITION(1),
```

Comment:
What are "STATS"? Position+length+BlockID? If yes, can we please use the word "reference"? Here and everywhere...

Reply:
Will do!